source/api_reference/version-history.html.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Version history
+weight: 0
+---
+# Version history
+
+## Version 1
+
+Version 1 can be accessed using either of these base URLs:
+
+- `https://{register-name}.register.gov.uk/v1/`
+- `https://{register-name}.register.gov.uk/`
+
+## Next version
```

Comment:
For style consistency it would be good to have spaces between headers and content.